src/main/java/nl/nn/testtool/TestTool.java

```diff
@@ -94,6 +95,12 @@ public class TestTool {
 	private @Setter @Getter @Inject @Autowired Views views;
 	private @Setter @Getter int reportsInProgressThreshold = 300000;
 	boolean devMode = false; // See testConcurrentLastEndpointAndFirstStartpointForSameCorrelationId()
+	private final OpenTelemetry openTelemetry;
+
+	@Autowired
+	TestTool (OpenTelemetry openTelemetry) {
```

Comment:
Why use a constructor? Can you change it to a setter?

Reply:
I removed the constructor